configure.ac

```diff
@@ -474,6 +474,17 @@ then LIBC=$withval
 else AC_MSG_ERROR([proper usage is --with-libc=STRING])
 fi])
 
+case $host in
+  dnl Windows path seperator
+  *-*-cygwin* | *-*-mingw* | *-*-msys*)
+    SH_PATH_SEP='\;'
+    ;;
+  *) dnl POSIX path seperator
+    SH_PATH_SEP=':'
+    ;;
+esac
+AC_SUBST(SH_PATH_SEP)
+
```

Comment:
Correct spelling is 'separator'. This replicates a lot of what JAVA_CLASSPATH_SEP does. Can you combine the two together and give it a better name, simply 'PATH_SEPARATOR'.

> SH_PATH_SEP='\;'

I don't understand why you need a backslash. Just use ";".

Reply:
I will fix spelling.

The backslash is the point.
On the testing we use bash and call:
`env <environment> ruby -Ipath1\;path2`
We tell bash to pass `-Ipath1;path2`  argument to ruby,
Without the backslash, we will have commands:
  `env <environment>  ruby -Ipath1` and `path2`.

Reply:
The real problem is you are missing quoting at the point of use:

```
  env LD_LIBRARY_PATH=.:$$LD_LIBRARY_PATH $(RUNTOOL) $(RUBY) $(RUBYFLAGS) -I$(srcdir)$(PATH_SEP). -- $(SCRIPTDIR)/
```

should be:

```
  env LD_LIBRARY_PATH=.:$$LD_LIBRARY_PATH $(RUNTOOL) $(RUBY) $(RUBYFLAGS) -I'$(srcdir)$(PATH_SEP).' -- $(SCRIPTDIR)/
```

Reply:
I'll try.
If it works than we can merge with Java separator.

Reply:
Cool, works.
I'll merge with Java separator.

Reply:
Is JAVA_CLASSPATH_SEP used?